.changeset/modern-hounds-give.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-auth-backend-module-bitbucket-provider': minor
+'@backstage/plugin-auth-backend': minor
```

Comment:
```suggestion
'@backstage/plugin-auth-backend': patch
```

No breaking change there, at least there shouldn't be 😁

Reply:
alright, I will take care of that, and then it will be just a patch change, yes.

Reply:
I have resolved both